src/site/content/en/learn/forms/design-basics/index.md

```diff
@@ -0,0 +1,215 @@
+  height: 450
+} %}
+
+### Help users navigate forms
+
+An extravagant layout can be fun, but may get in the way of completing a form.
+
+In particular,
+[research shows](https://baymard.com/blog/avoid-multi-column-forms)
+that it's best to use only a single column.
+Users don't want to spend time searching where the next form control is.
+By using one column, there is only one direction to follow.
+
+{% Aside %}
+Always ensure that forms are accessible for people who use only the keyboard to navigate websites—either through preference or circumstance.
+Test your form by only using keyboard navigation.
+The tab order should follow the visual order.
+You can view the
+[source order in Chrome DevTools](https://developer.chrome.com/blog/new-in-devtools-92/#source-order).
+{% endAside %}
+
+### Help users interact with forms
+
+To avoid accidental taps and clicks,
+and help users interact with your form, make your buttons big enough.
+The recommended
+[tap target size](/accessible-tap-targets/) of a button is at least 48px.
+You should also add enough spacing between form controls using the `margin`
+CSS property to help avoid accidental interactions.
+
+The default size of form controls is too small to be really usable. You should increase the size by using `padding` and changing the default `font-size`.
+
+{% Aside %}
+Use at least `1rem` (which for most browsers has a default value of 16px)
+as the `font-size` for your form controls.
+This [prevents a page zoom](https://css-tricks.com/16px-or-larger-text-prevents-ios-form-zoom/)
+on iOS Safari when a form control is focused.
+{% endAside %}
+
+You can define different sizes for different pointing devices,
+for example, a mouse, using the  `pointer` CSS media feature.
+
+```css
+// has a pointing device, for example, a mouse
+@media (pointer: fine) {
+  input[type="checkbox"] {
+    width: 15px;
+    height: 15px;
+  }
+}
+
+// pointing device of limited accuracy, for example, a touch-based device
+@media (pointer: coarse) {
+  input[type="checkbox"] {
+    width: 30px;
+    height: 30px;
+  }
+}
+```
+
+Learn more about the
+[`pointer` CSS media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer).
+
+### Show errors where they happen
+
+To make it straightforward for users to find out which form control needs their attention,
+display error messages next to the form control they refer to.
+When displaying errors on form submission, make sure to navigate to the first invalid form control.
+
+### How does your user know what data to enter?
+
+Make sure users understand how to enter valid data.
+Do they need to enter at least eight characters? Tell them.
+
+```html
+<label for="password">Password (required)</label>
+<input required minlength="8" type="password" id="password"
+  name="password" aria-describedby="password-minlength">
+<span id="password-minlength">Enter at least eight characters</span>
+```
+
+{% Aside %}
+Make it clear to users which fields are required.
```

Comment:
Not sure, why I added this inside an aside, I think we don't need the aside here.

```suggestion
### Make it clear to users which fields are required.